### ISSUE TITLE
Inconsistent output behavior of `mindspore.nn.LPPool1d` compared to PyTorch for negative inputs and fractional `norm_type`

### ISSUE THREAD

**Issue by tingPetty** [mindspore-ai/mindspore#349]
### Hardware Environment(`Ascend`/`GPU`/`CPU`):

/device cpu

### Software Environment:

- **MindSpore version (source or binary)**: 2.5.0 (binary)
- **Python version**: 3.10.18
- **OS platform and distribution**: Windows 11
- **GCC/Compiler version**: N/A

## Describe the current behavior

When computing `LPPool1d` with `norm_type=1.5` on an input tensor containing negative values, `mindspore.nn.LPPool1d` outputs `0.0`.

## Describe the expected behavior

The expected behavior should either explicitly raise an error or output `NaN` (as PyTorch does), since raising a negative number to a fractional power (1.5) is undefined in the real number domain. The current output of `0.0` masks potential numerical errors.

## Steps to reproduce the issue

1. Define an input numpy array containing negative float values.
2. Initialize `mindspore.nn.LPPool1d(norm_type=1.5, kernel_size=3, stride=1)`.
3. Pass the tensor into the operator and observe the output.

Python

```
import numpy as np
import torch
import mindspore
import mindspore.nn as ms_nn

# Sample: llm_enhanced_torch_nn_LPPool1d_20251215_193350.json_sample1.txt
input_np = np.array(
    [
        0.63573098,
        0.81810886,
        -0.96552032,
        -0.99548149,
        0.22021013,
        -0.02396944,
    ],
    dtype=np.float32,
).reshape(2, 1, 3)

# PyTorch
pt_layer = torch.nn.LPPool1d(norm_type=1.5, kernel_size=3, stride=1)
out_pt = pt_layer(torch.tensor(input_np)).detach().numpy()

# MindSpore
mindspore.set_context(mode=mindspore.PYNATIVE_MODE)
ms_layer = ms_nn.LPPool1d(norm_type=1.5, kernel_size=3, stride=1)
out_ms = ms_layer(mindspore.Tensor(input_np)).asnumpy()

abs_diff = np.abs(out_pt - out_ms)
max_diff = np.nan if np.isnan(abs_diff).all() else np.nanmax(abs_diff)
print(f"PyTorch output: {out_pt}")
print(f"MindSpore output: {out_ms}")
print(f"NaN count (PT, MS): {np.isnan(out_pt).sum()}, {np.isnan(out_ms).sum()}")
print(f"Maximum difference: {max_diff}")
```

## Related log / screenshot

Plaintext

```
PyTorch output: [[[nan]]

 [[nan]]]
MindSpore output: [[[0.]]

 [[0.]]]
NaN count (PT, MS): 2, 0
Maximum difference: nan
```
